package.json

```diff
@@ -70,6 +70,11 @@
 					"type": "boolean",
 					"default": true,
 					"description": "Forces indenting with two spaces when Dart files are opened. This is on by default because VS Code doesn't support per-language settings and most people use tabs/4 spaces for other languages."
+				},
+				"dart.showTodos": {
+					"type": "boolean",
+					"default": false,
```

Comment:
> Let's leave as is for now

I think you're right about the discovery issue - people won't know to turn it on.

That todo extension looks nice. It does seem like you want to work with todos occasionally, and not all the time like errors or warnings.